src/launcher/standalone_launcher/.env.production

```diff
@@ -0,0 +1 @@
+REACT_APP_GREENKEY_URL=wss://web-beta.adaptivecluster.com/client/ws/speech
```

Comment:
Add newline at end of file. (Please set up your IDE to insert these automatically).